Bottom: c974b5f93505ebebeab7215ab480e8fa62d6a223
Top:    d0077a3c6fcf24f404f72500184ce2e0d7277625
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 10:41:43 +0100

Refresh of add-a-demonstration-model-for-

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d6010b8..fd06e90 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2188,7 +2188,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   /* Scale board coordiantes to (-1,-1)-(1,1) coordiantes */
   /* Adjust the "w" coordinate of our homogeneous coodinates. We coulld in
    * theory just use glScalef to transform, but on mesa this produces errors
-   * as the resulting modelview matrix has a very smalldeterminant.
+   * as the resulting modelview matrix has a very small determinant.
    */
   scale[15] = (float)port->zoom * (float)MIN (widget->allocation.width, widget->allocation.height) / 2.;
   /* XXX: Need to choose which to use (width or height) based on the aspect of the window
